test/lib/docker.test.ts

```diff
@@ -253,13 +253,77 @@ test("findGlobs", async (t) => {
     t.same(files, ["/pom.xml", "/dir3/file6.csproj"]);
   });
 
-  t.test("find globs using registry globs and fixture", async (t) => {
+  t.test(
+    "find globs on ghost app using node_modules exclude glob",
+    async (t) => {
+      stub.resolves({
+        stdout: readFileSync(getLSOutputFixture("ghost-app.txt")).toString(),
+      });
+
+      const files = await docker.findGlobs(
+        ["**/package.json", "**/package-lock.json", "**/yarn.lock"],
+        ["**/node_modules/**"],
+      );
+
+      t.same(files, [
+        "/opt/yarn-v1.16.0/package.json",
+        "/var/lib/ghost/versions/2.25.6/package.json",
+        "/var/lib/ghost/versions/2.25.6/yarn.lock",
+        "/var/lib/ghost/versions/2.25.6/content/themes/casper/package.json",
+        "/var/lib/ghost/versions/2.25.6/content/themes/casper/yarn.lock",
+      ]);
+    },
+  );
+
+  t.test("find globs on alpine", async (t) => {
     stub.resolves({
       stdout: readFileSync(
         getLSOutputFixture("alpine-3.9.4-manifest-files.txt"),
       ).toString(),
     });
-    const files = await docker.findGlobs(registryGlobs);
+
+    const files = await docker.findGlobs([
+      "**/package.json",
+      "**/Gemfile.lock",
+    ]);
+
     t.same(files, ["/app/Gemfile.lock", "/srv/app/package.json"]);
   });
+
+  t.test("find globs on centos", async (t) => {
+    stub.resolves({
+      stdout: readFileSync(
+        getLSOutputFixture("centos-7.6.1810-manifest-files.txt"),
+      ).toString(),
+    });
+
+    const files = await docker.findGlobs(["**/pom.xml"]);
+
+    t.same(files, ["/app/pom.xml"]);
+  });
+
+  t.test("find globs on centos", async (t) => {
+    stub.resolves({
+      stdout: readFileSync(
+        getLSOutputFixture("debian-10.0-manifest-files.txt"),
+      ).toString(),
+    });
+
+    const files = await docker.findGlobs(["**/package.json"]);
+
+    t.same(files, ["/app/package.json"]);
+  });
+
+  t.test("find globs on centos", async (t) => {
```

Comment:
I suggest naming this test differently than the previous https://github.com/snyk/snyk-docker-plugin/pull/101/files#diff-36abe35061e760a3bbe9c05b3d215723R305
specifically with a verbose name that differentiates it with its purpose and test subject.

Reply:
oh dear, yes the names are very wrong! Also here I am testing Ubuntu and not Centos!